composer.json

```diff
@@ -14,7 +14,9 @@
         "aura/di": "dev-develop-2",
         "bodom78/kohana-imagefly": "dev-master",
         "league/oauth2-server": "~3.2.4",
-        "league/flysystem": "0.4.*",
+        "league/flysystem-aws-s3-v3": "1.0.4",
```

Comment:
Agreed. Drop PHP 5.4 .. I added the php version requirement to composer too, so make sure to update that. And run composer update on php 5.5